libr/bin/format/elf/elf.c

```diff
@@ -3802,9 +3785,9 @@ ut64 Elf_(r_bin_elf_p2v) (ELFOBJ *bin, ut64 paddr) {
 
 /* Deprecated temporarily. Use r_bin_elf_v2p_new in new code for now. */
 ut64 Elf_(r_bin_elf_v2p) (ELFOBJ *bin, ut64 vaddr) {
-	int i;
+	size_t i;
 
-	r_return_val_if_fail (bin, 0);
+	r_return_val_if_fail (bin, 0); // UT64_MAX or vaddr?
```

Comment:
no need for this comment. This function shall be removed at some point. As it can be seen, the `_new` version shall be used for all future code.

Reply:
this must be added into the elf-ut64-max pr